packages/vscode-extension/src/exp/index.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as vscode from "vscode";
+import { IExperimentationService, IExperimentationTelemetry, TargetPopulation, getExperimentationServiceAsync } from "vscode-tas-client";
+import * as pkg from "./../../package.json";
+import { ExtTelemetry } from "./../telemetry/extTelemetry"
+
+class ExperimentationTelemetry implements IExperimentationTelemetry {
+    private sharedProperties: {[key: string]: string};
+
+    constructor() {
+        this.sharedProperties = {};
+    }
+
+    public setSharedProperty(name: string, value: string): void {
+        this.sharedProperties[name] = value;
+    }
+
+    public postEvent(eventName: string, props: Map<string, string>): void {
+        let properties = { ...this.sharedProperties };
+        props.forEach((value, key) => {
+            properties[key] = value;
+        });
+        ExtTelemetry.sendTelemetryEvent(eventName, properties);
```

Comment:
does this mean Exp has a series of its own defined telemetries?

Reply:
Yes, here is an example:
![image](https://user-images.githubusercontent.com/71362691/122517847-15ad5780-d043-11eb-91a2-c1ef05af1b44.png)